pkgs/applications/video/obs-studio/default.nix

```diff
@@ -20,7 +20,7 @@
 , pkgconfig
 , vlc
 , mbedtls
-
+, wrapQtAppsHook
```

Comment:
Turns out we probably want to use `mkDerivation` from the qt5 set, instead of manually handling the wrapping. See the other comment about that.

(I need a comment to ask for changes :/)

Reply:
Yeah, turns out a lot of the qt applications in nixpkgs aren't packaged correctly at all...